src/shims/time.rs

```diff
@@ -159,4 +160,25 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         u64::try_from(duration.as_nanos())
             .map_err(|_| err_unsup_format!("programs running longer than 2^64 nanoseconds are not supported").into())
     }
+
+    fn mach_timebase_info(&mut self, info_op: OpTy<'tcx, Tag>) -> InterpResult<'tcx, i32> {
+        let this = self.eval_context_mut();
+
+        this.assert_target_os("macos", "mach_timebase_info");
+        this.check_no_isolation("mach_timebase_info")?;
+
+        let info = this.deref_operand(info_op)?;
+
+        // Since we return nanoseconds instead of ticks from
+        // `mach_absolute_time`, we don't need to scale the absolute
+        // time.
```

Comment:
I just realized I am not sure what "absolute" refers to here. This is still a monotone clock that cannot be converted to "absolute" wall-clock time.
```suggestion
        // Since our emulated ticks in `mach_absolute_time` *are* nanoseconds,
        // no scaling needs to happen.
```

Reply:
That is much clearer. I've committed the suggestion.